src/components/Display.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
+
+const DisplayLayout = styled.div`
```

Comment:
This component shouldn't be changed in this PR. Sorry for the misunderstanding :)